src/commands/folder/spec/folder.spec.server.ts

```diff
@@ -0,0 +1,191 @@
+import dotenv from 'dotenv'
+import { folder } from '../index'
+import { generateTimestamp } from '../../../utils/utils'
+import { ServerType, Target } from '@sasjs/utils/types'
+import {
+  removeFromGlobalConfig,
+  saveToGlobalConfig
+} from '../../../utils/config'
+import { Command } from '../../../utils/command'
+
+describe('sasjs job execute', () => {
+  let target: Target
+
+  beforeAll(async (done) => {
+    target = await createGlobalTarget()
+    process.projectDir = process.cwd()
+
+    done()
+  })
+
+  afterAll(async (done) => {
+    await removeFromGlobalConfig(target.name)
+    done()
+  })
+
+  it('list folder children', async (done) => {
+    const timestamp = generateTimestamp()
+    const testFolderPath = `/Public/app/cli-tests/cli-tests-folder-${timestamp}`
+
+    await createTestFolder(testFolderPath, target.name)
+
+    const commandList1 = new Command(
+      `folder list /Public/app/cli-tests -t ${target.name}`
+    )
+    const list1 = await folder(commandList1)
+    expect(list1).toContain(`cli-tests-folder-${timestamp}`)
```

Comment:
it would be better to use RegExp to verify that `cli-tests-folder-${timestamp}` entry is in the right place